templates/confluence.service.erb

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Confluence Team Collaboration Software
+After=network.target
+
+[Service]
+Type=forking
+User=confluence
+PIDFile=<%= scope.lookupvar('confluence::webappdir') %>/work/catalina.pid
```

Comment:
I did misunderstand, my apologies. And you are correct, the PIDFile isn't necessary. I've removed it and pushed a new commit.